src/fadec/src/EngineControl.h

```diff
@@ -20,11 +20,11 @@
 
 /* Values in gallons */
 struct Configuration {
-    double fuelCenter = 0;
```

Comment:
Looks like VSCode applied formatting in some places.. Maybe makes sense at some places but I can revert that if needed.